tests/test_edge_cases.py

```diff
@@ -167,6 +167,8 @@ def test_lfc_shrinkage_coeff():
 
     res = DeseqStats(dds)
     res.summary()
+    res.lfc_shrink(coeff=None)
```

Comment:
would it be worth adding a test for other coeff values?

Reply:
I've updated the test to check that the same result is obtained with the implicit coeff `condition_B_vs_A` (the case where an incorrect coeff is provided is already covered)